build.gradle.kts

```diff
@@ -5,6 +5,7 @@ buildscript {
         google()
         mavenCentral()
         mavenLocal()
+        maven("https://oss.jfrog.org/artifactory/oss-release-local")
```

Comment:
I assume we use this repo for `build-info-extractor-gradle` ?
I have no idea why we use that plugin, can we maybe add a comment here to say that this line is needed for that plugin? That way we removed them altogether if we find out that it is not needed.